backend/isic/README.md

```diff
@@ -0,0 +1,29 @@
+# Instruction
+To use the module from terminal,run:
+
+```console
+$ python isic-crawler.py --numjson $val1 --numimage $val2 --dermo $bool1 --melano $bool2

```

Comment:
```suggestion
```bash
python isic-crawler.py --numjson $val1 --numimage $val2 --dermo $bool1 --melano $bool2
```

Reply:
I cut off this path already as I remove the main function,but thank you